### ISSUE TITLE
[a11y]: New message in ai-chat doesn't have alert and is not read out

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#373]
### Browser

Firefox

### Operating System

Windows

### Package version

0.14.1-canary.347

### React version

^1.66.0

### Automated testing tool and ruleset

none

### Assistive technology

JAWS 2025

### Description

For the ai-chat component, when a new message is returned based on the user-initiated question, the user is not alerted to a new response and it is not read out automatically for them.

Test criteria:
Testers should pay particular attention that the **speech output of the screen reader matches the visual information and that messages and dynamic updates are properly detected and announced by the screen reader**. Testers may wish to inspect code to verify if properties, relationships, and states are properly set. Also check that messages and time limit notifications that appear on screen and do not take focus are still read by the screen reader depending on the verbosity settings.

One can use the up arrow to read out the response message, but the issue is that a non-sighted user won't be aware a new message came in so they won't know to use the arrow to read the message. So I think we're looking at two options:

- There is an alert that the new message came in for the user to go read it
- Ideally, the message is just read aloud as it is in Chrome when it comes in

### Reproduction/example

none

### Steps to reproduce

1. leverage the ai-chat component
2. enter a chat message
3. wait for a response

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
